docs/serving/configuration/deployment.md

```diff
@@ -106,3 +109,25 @@ data:
   # List of repositories for which tag to digest resolving should be skipped
   registries-skipping-tag-resolving: registry.example.com
 ```
+
+## Enable container-freezer service
+
+You can configure queue-proxy to pause pods when not in use by enabling the `container-freezer` service. It calls a stand-alone service (via a user-specified endpoint) when a pod's traffic drops to zero or scales up from zero. To enable it, set `concurrency-state-endpoint` to a non-empty value. With this configuration, you can achieve some features like freezing running processes in pods or billing based on the time it takes to process the requests.
+
+Before you configure this, you need to implement the endpoint API. The official implementation is [container-freezer](https://github.com/knative-sandbox/container-freezer).
```

Comment:
```suggestion
Before you configure this, you need to implement the endpoint API by following the installation instructions in the [container-freezer README](https://github.com/knative-sandbox/container-freezer).
```

Reply:
Technically, and this is very much a nit, users do have the option of implementing their own service (as long as it could handle the endpoint calls) and use that instead of the container-freezer. So the freezer isn't strictly necessary, although it's probably what we want to recommend. Not sure exactly the best way to phrase that for the docs though.

Reply:
That wording sounds fine to me, but I would keep the first sentence the same as it was before because we prefer the to use active voice so it is clear who is implementing the endpoint API. Also tweak the location of the link:

```suggestion
Before you configure this, you need to implement the endpoint API. The official implementation is container-freezer. You can install it by following the installation instructions in the [container-freezer README](https://github.com/knative-sandbox/container-freezer).
```